q2_feature_classifier/citations.bib

```diff
@@ -0,0 +1,41 @@
+% TODO: Update this to the final publication ASAP
```

Comment:
@nbokulich may be able to get an actual in-press citation soon

Reply:
he said he's going to do that, so i'll do a last review once that one is in.

Reply:
@ebolyen I've added the in-press citation to your branch in [this PR](https://github.com/ebolyen/q2-feature-classifier/pull/1)

Seems like we won't be able to get a DOI before release time... so we can unblock this once you merge my PR.